src/sagemaker/workflow/pipeline.py

```diff
@@ -912,6 +947,25 @@ class _ExecutionList:
     pipeline_executions: Sequence[_PipelineExecution] = attr.ib(factory=list)
     next_token: str = attr.ib(default=None)
 
+    def table(self):
+        """Displays the list of _PipelineExecution objects as a Table
+
+        Returns:
+            A table visual of the list of _PipelineExecution objects
+        """
+        input_list = self.pipeline_executions
+        next_token = self.next_token
+
+        pd.set_option("display.max_colwidth", None)
+        df = pd.DataFrame.from_records([each.to_dict() for each in input_list])
```

Comment:
would the existing `to_request` work here? if so please add TODO comment for the replacement

Reply:
the existing to_request does not correctly convert certain attributes like the Parameters into an output string, so that is why I added .to_dict(). However, this may be worth looking into more so I will add to TODO

Reply:
Sorry, after a second thought, let's keep the to_dict. As to_request contains information we don't need